third_party/mbedtls/hardware_entropy.c

```diff
@@ -1,5 +1,5 @@
 /*
- *    Copyright 2016 Nest Labs Inc. All Rights Reserved.
+ *    Copyright 2016 The OpenThread Authors. All Rights Reserved.
```

Comment:
We'll follow-up. It is up to the contributor or copyright owner on how code in third_party is licensed, so I think it *may* be under The OpenThread Authors if we choose.